openhtf/util/xmlrpcutil.py

```diff
@@ -16,13 +16,20 @@
 
 import http.client
 import xmlrpc.server
+import os
 import socketserver
+import sys
 import threading
 import xmlrpc.client
 import collections
 
 DEFAULT_PROXY_TIMEOUT_S = 3
 
+if os.name == 'posix' and sys.version_info[0] < 3:
```

Comment:
`future` takes care of this without having to detect which Python version:

http://python-future.org/compatible_idioms.html#xmlrpc-module

Reply:
Also I'm not sure what the os.name == 'posix' clause guards against, do these packages not work on windows or something?